plugins/ruby-foreman-docker/debian/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-foreman-docker (1.4.0-1) stable; urgency=low
+
+  * Update release to 1.4.0
+
+ -- Daniel Lobato Garcia <elobatocs@gmail.com>  Mon, Jul 27 2015 12:04:44 -0400
```

Comment:
I had to fix an error here, the day and month are the wrong way around.  Please use the provided helper, scripts/changelog.rb or dch.